flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/KinesisDataFetcher.java

```diff
@@ -326,7 +326,7 @@ public RecordQueue<RecordWrapper<T>> getQueue(int producerIndex) {
             return queues.computeIfAbsent(
                     producerIndex,
                     (key) ->
-                            new RecordQueue<RecordWrapper<T>>() {
+                            new RecordEmitter.RecordQueue<RecordWrapper<T>>() {
```

Comment:
nit: This is unnecessary

Reply:
Apologies, this snuck in there by accident. Removed it